### PR TITLE
Previews for pictures that displayed on the screen instead of all.

### PR DESCRIPTION
**Pull request by SLemishko**
Additionally add preview of the pictures in search mode.

https://trello.com/c/cMevJitP/16244-speed-up-filebrowser-for-large-partitions?filter=label:Release%202.5

### CODE REVIEW

**Comment by SLemishko** [comment]
Before-After-Search

File: lib/material/internal/filebrowser/material_filebrowser_internal_view.flow [hunk 1]
```diff
@@ -253,25 +253,24 @@ createContentItem(state : FilebrowserState<?, ??, ???>, itemCP : FBContentPath,
 				newCPItem = FBContentPath(itemItem, itemCP.propertiesR, itemCP.isFolder, itemCP.dirPath, state.item2name(itemItem));
 				FBContentItem(
 					newCPItem,
-					createItemUI(state, newCPItem, changeItem, fif(state.contentViewIconsB, previewsB, const([])))
+					createItemUI(state, newCPItem, changeItem, requestPreview)
```
Comment (NikitaFil) [inline lib/material/internal/filebrowser/material_filebrowser_internal_view.flow:256]: state.contentViewIconsB is now unsupported

**Review by NikitaFil — CHANGES_REQUESTED** [review]
Generally looks fine, but one more fix is needed.

**Review by NikitaFil — APPROVED** [review]

**Review by MalkoVl — COMMENTED** [review]
Some explanation are required.

**Review by MalkoVl — APPROVED** [review]

**Pull request merged**